src/boost.mk

```diff
@@ -69,14 +71,14 @@ define $(PKG)_BUILD
     echo 'set(Boost_THREADAPI "win32")' > '$(CMAKE_TOOLCHAIN_DIR)/$(PKG).cmake'
 
     '$(TARGET)-g++' \
-        -W -Wall -Werror -ansi -pedantic \
+        -W -Wall -Werror -ansi -pedantic -std=c++11 \
         '$(PWD)/src/$(PKG)-test.cpp' -o '$(PREFIX)/$(TARGET)/bin/test-boost.exe' \
         -DBOOST_THREAD_USE_LIB \
-        -lboost_serialization-mt \
-        -lboost_thread_win32-mt \
-        -lboost_system-mt \
-        -lboost_chrono-mt \
-        -lboost_context-mt
+        -lboost_serialization$($(PKG)_SUFFIX) \
+        -lboost_thread$($(PKG)_SUFFIX) \
+        -lboost_system$($(PKG)_SUFFIX) \
+        -lboost_chrono$($(PKG)_SUFFIX) \
+        -lboost_context$($(PKG)_SUFFIX)
```

Comment:
Could you explain this change? I don't think I understand where $(PKG)_SUFFIX is set.

Reply:
The library names were changed from just `-mt` to `-mt-x32` or `-mt-x64`, depends on the target platform.